avaframe/in1Data/computeFromDistribution.py

```diff
@@ -7,6 +7,8 @@
 import scipy.special as sc
 from scipy.interpolate import interp1d
 import logging
+from scipy.stats import norm
+import matplotlib.pyplot as plt
```

Comment:
Please remove this import. It is not needed, right?

Reply:
removed